docs/book/src/reference/strict-field-validation.md

```diff
@@ -0,0 +1,214 @@
+# Strict Field Validation
```

Comment:
Sorry was just quickly reading over it so I might have missed this. This seems to focus only on CRDs?

If strict validation is used with builtin-resources this can mean that the controller is only compatible with exactly the Kubernetes version it was compiled against, which is often not desired